LICENSE.txt

```diff
@@ -1,23 +1,683 @@
-LokiNET is the reference implementation of LLARP (Low Latency Anonymous 
-Routing Protocol) and is licensed under ZLIB license, however the protocol 
-specifications are placed into the public domian using the CC0 License.
+LokiNET is the reference implementation of LLARP (Low Latency Anonymous
+Routing Protocol); this distribution of LokiNET and the Lokinet control
+panel is licensed under the GNU General Public License, version 3.
```

Comment:
You don't want this control panel stuff; that was there just for the control panel itself.  Instead:

`is the reference implementation of LLARP (Low Latency Anonymous
Routing Protocol).  LokiNET is licensed under the GNU General Public License, version 3.`

Reply:
Actually just using the GNU-recommended preamble might be better:

    LokiNET is the reference implementation of LLARP (Low Latency Anonymous 
Routing Protocol).

    This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

(Do we want the "or later" part?)